frontend/src/core/app/helpers.js

```diff
@@ -190,6 +186,14 @@ define(function(require){
 
         },
 
+        ifImageIsCourseAsset: function(url, block) {
```

Comment:
This doesn't modify anything though does it?

Given: 
```
{{#customIf}}
    <div class="ifDiv"></div>
{{else}}
    <div class="elseDiv"></div>
{{/customIf}}
```
Retuning `block.fn(this)` will insert the `ifDiv`, whereas returning `block.inverse(this)` will insert the `elseDiv` (as I understand it).